### ISSUE TITLE
VS Code extensions marketplace

### ISSUE THREAD

**Issue by Wgmlgz** [mamsdeveloper/bcomp-asm#1]
Will you publish your extension to vscode extensions marketplace?
(Please do 👉👈)

**Comment by butvinm** [comment]
Maybe I`ll do it later, but now you can install it with .vsix file from release/ folder, I give instruction in readme

**Comment by Wgmlgz** [comment]
Yep, I understand, I have my own extension too. But it feels kinda incomplete, if you have to install it manually. Besides that, if there is an update, you have to reinstall it again, and since this project is open source and very useful for many people, I believe that there will be some updates.
